ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -373,13 +384,15 @@ const getValueDisplay = (pop, valueField, precision) => (valueField === 'ac' ?
   `${pop.ac} out of ${pop.an}` : `${pop[valueField].toPrecision(precision || 2)}`)
 
 const Frequencies = React.memo(({ variant }) => {
-  const { populations = {} } = variant
+  const { populations = {}, svType } = variant
   const callsetHetPop = populations.callset_heteroplasmy || populations.seqr_heteroplasmy
   const isMito = callsetHetPop && callsetHetPop.af !== null && callsetHetPop.af !== undefined
   const popConfigs = isMito ? MITO_POPULATIONS : POPULATIONS
+  let datasetType = isMito ? DATASET_TYPE_MITO_CALLS : DATASET_TYPE_SNV_INDEL_CALLS
+  datasetType = svType ? DATASET_TYPE_SV_CALLS : datasetType
   const seqrAcSection = {
     name: 'seqr Global ACs',
-    details: (!isMito && populations[SEQR_POP.field]) ? [<GlobalAcPopup />].filter(s => s) : [],
+    details: populations[SEQR_POP.field] ? [<GlobalAcPopup datasetType={datasetType} />].filter(s => s) : [],
```

Comment:
when I did the initial global AF deprecation I only did it for SNV_INDEL - this updates the UI to work with any dataset type